src/components/Standings/place.js

```diff
@@ -16,7 +16,15 @@ class Place extends React.Component {
     }
     return (
       // Renders the individual row of the leaderboard for each team
-      <View style={styles.row}>
+      <View style={
+        this.props.showScoresByTeam
+          ? this.props.teamNumber == this.props.userTeamNumber 
+            ? styles.highlightedRow 
+            : styles.row
+          : this.props.uid == this.props.userID
```

Comment:
I would probably move these logic operations to the main Standings file and instead have a boolean property that is called `isHighlighted` for the place component that highlights it or not. This way we can choose to highlight dynamically in the future if we need to.